packages/utils/plugins-helpers/src/types.ts

```diff
@@ -218,6 +218,14 @@ export namespace Types {
   export type NamedPreset = string;
   export type OutputConfig = NamedPlugin | ConfiguredPlugin;
 
+  export type PresetNamesBase =
```

Comment:
We will improve this in the future by inferring the preset options from the preset function's typings (or another build-based approach)

Reply:
Ideally, we should aim to infer that options/config and flags based on the preset/plugins name (in TS files). 
But that's for later